whycpp_c/src/log.cpp

```diff
@@ -0,0 +1,22 @@
+#include <whycpp_c/log.h>
+#include <SDL2/SDL_error.h>
```

Comment:
this file and entire whycpp_c wrapper should be better not to depend on SDL2 OR we need to configure cmake to include SDL2 headers for this project.
Is there a way to remove this includes? Maybe we need to add some function to the root-whycpp?